lib/Models/Catalog/Ows/WebMapServiceCatalogItem.ts

```diff
@@ -511,18 +511,6 @@ class WebMapServiceCatalogItem
         ? this.diffModeParameters
         : {};
 
-      // Set CRS for WMS 1.3.0
-      // Set SRS for WMS 1.1.1
-      if (this.crs) {
-        if (this.useWmsVersion130) {
-          parameters.crs = this.crs;
-          getFeatureInfoParameters.crs = this.crs;
-        } else {
-          parameters.srs = this.crs;
-          getFeatureInfoParameters.srs = this.crs;
-        }
-      }
-
```

Comment:
Instead of passing `crs` `srs` as `parameters`, pass them as constructor options to `WebMapServiceImageryProvider` below.